sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -318,18 +318,53 @@ export function isJSONLikeObject(value: any): boolean {
  * @param value
  */
 export function getMessageCountDetails(value: any): MessageCountDetails {
+  const xmlnsPrefix = getXMLNSPrefix(value);
   if (value == undefined) {
     value = {};
   }
   return {
-    activeMessageCount: parseInt(value["d2p1:ActiveMessageCount"]) || 0,
-    deadLetterMessageCount: parseInt(value["d2p1:DeadLetterMessageCount"]) || 0,
-    scheduledMessageCount: parseInt(value["d2p1:ScheduledMessageCount"]) || 0,
-    transferMessageCount: parseInt(value["d2p1:TransferMessageCount"]) || 0,
-    transferDeadLetterMessageCount: parseInt(value["d2p1:TransferDeadLetterMessageCount"]) || 0
+    activeMessageCount: parseInt(value[`${xmlnsPrefix}:ActiveMessageCount`]) || 0,
+    deadLetterMessageCount: parseInt(value[`${xmlnsPrefix}:DeadLetterMessageCount`]) || 0,
+    scheduledMessageCount: parseInt(value[`${xmlnsPrefix}:ScheduledMessageCount`]) || 0,
+    transferMessageCount: parseInt(value[`${xmlnsPrefix}:TransferMessageCount`]) || 0,
+    transferDeadLetterMessageCount:
+      parseInt(value[`${xmlnsPrefix}:TransferDeadLetterMessageCount`]) || 0
   };
 }
 
+/**
+ * @internal
+ * @hidden
+ * Gets the xmlns prefix from the root of the objects that are part of the parsed response body.
+ */
+export function getXMLNSPrefix(value: any) {
+  if (!value[Constants.XML_METADATA_MARKER]) {
```

Comment:
Does this function work in browsers as well?

Reply:
It should.
It does.

This function just gets the xmlns prefix from an object such as below.
```js
{
          $: {
            "xmlns:d3p1": "http://schemas.microsoft.com/netservices/2011/06/servicebus"
          },
          "d3p1:ActiveMessageCount": "3",
          "d3p1:DeadLetterMessageCount": "0",
          "d3p1:ScheduledMessageCount": "0",
          "d3p1:TransferMessageCount": "0",
          "d3p1:TransferDeadLetterMessageCount": "0"
}
```